test/compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+services:
+  switchboard:
+    image: switchboard/switchboard
+    ports: 
+      - 8080:8080
+    volumes: 
+      - toolsdata:/switchboard-tool-registry:ro
+    logging:
+      driver: "none"
```

Comment:
Shouldn't we log everything during the tests?

Reply:
I added this because the switchboard output is usually not interesting, but I can take it out.

Reply:
Yeah, but it also displays all the http requests that cypress does, and that makes more difficult to see the cypress results. I should add the logs back, because the actual debugging will not happen in the CI but on our own machines, this is just to make sure all the tests are working.